include/flatbuffers/reflection_generated.h

```diff
@@ -6,6 +6,12 @@
 
 #include "flatbuffers/flatbuffers.h"
 
+#if defined(_MSC_VER)
```

Comment:
maybe compare against the actual version of visual studio to make sure it work on later version, e.g. http://stackoverflow.com/questions/18387640/how-to-deal-with-noexcept-in-visual-studio